### PR TITLE
Add Tina Quach to Credits

### PR DESCRIPTION
**Pull request by GreenBayRules**
Solves #1572

### CODE REVIEW

File: src/views/credits/credits.jsx [hunk 1]
```diff
@@ -71,7 +71,12 @@ var Credits = React.createClass({
                         <img src="//cdn.scratch.mit.edu/get_image/user/2286560_170x170.png" alt="Carmelo Avatar" />
                         <span className="name">Carmelo Presicce</span>
                     </li>
-
+                    
+                    <li>
+                         <img src="//cdn2.scratch.mit.edu/get_image/user/default_170x170.png" alt="Tina Avatar" />
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Thanks! Tina has set an avatar. Can you update this to reflect that?

Comment (Kenny2github) [reply src/views/credits/credits.jsx]: Might need the URL unless there's an easy way to find that.

**Comment by GreenBayRules** [comment]
Okay it should be ready now.

File: src/views/credits/credits.jsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ var Credits = React.createClass({
                     </li>
                     
                     <li>
-                         <img src="//cdn2.scratch.mit.edu/get_image/user/default_170x170.png" alt="Tina Avatar" />
+                         <img src="https://cdn2.scratch.mit.edu/get_image/user/25500116_170x170.png" alt="Tina Avatar" />
```
Comment (thisandagain) [inline src/views/credits/credits.jsx]: Can you remove the `https:` (see other lines)? Otherwise this looks good!

Comment (GreenBayRules) [reply src/views/credits/credits.jsx]: yep ookay

**Comment by GreenBayRules** [comment]
Removed https://

**Comment by thisandagain** [comment]
Thanks @GreenBayRules !

**Comment by mewtaylor** [comment]
Awesome congrats on your first www PR @GreenBayRules 
confetti-horn-emoji

**Comment by GreenBayRules** [comment]
Thanks :D Working for a fix for another issue too :D

**Comment by mewtaylor** [comment]
Testing on staging, Chrome 61, OSX 10.12:
* On `/info/credits`, "Tina Quach" shows up on the page, with the correct avatar

Marking this as `QA-OK`

**Pull request merged**